how-to-docs/define-rbac-policy.md

```diff
@@ -0,0 +1,37 @@
+# How to define a new RBAC policy
+
+Visit and clone the [acp-kube-rbac][acp kube rbac repo link] repo on GitLab.
+
+Create a new branch for your new policy and follow the instructions on the README in the repo.
+
+Documentation that can help write an RBAC policy can be found here: [RBAC Authorization][rbac docs].
+
+ * Please note that the docs linked above are only for reference.
```

Comment:
This looks like a note not a bullet point so might be best to format it as ```note: Please note that the docs linked above are only for reference.``` or something similear